stored_responses/stored_responses_test.go

```diff
@@ -581,17 +661,19 @@ func TestProcessStoredAuctionAndBidResponses(t *testing.T) {
 	}
 
 	for _, test := range testCases {
-		storedAuctionResponses, storedBidResponses, bidderImpReplaceImpId, errorList := ProcessStoredResponses(nil, test.requestJson, fetcher, bidderMap)
-		assert.Equal(t, test.expectedStoredAuctionResponses, storedAuctionResponses, "storedAuctionResponses doesn't match: %s\n", test.description)
-		assert.Equalf(t, test.expectedStoredBidResponses, storedBidResponses, "storedBidResponses doesn't match: %s\n", test.description)
-		assert.Equal(t, test.expectedBidderImpReplaceImpID, bidderImpReplaceImpId, "bidderImpReplaceImpId doesn't match: %s\n", test.description)
-		assert.Nil(t, errorList, "Error should be nil")
+		t.Run(test.description, func(t *testing.T) {
+			rw := openrtb_ext.RequestWrapper{BidRequest: &test.request}
+			storedAuctionResponses, storedBidResponses, bidderImpReplaceImpId, errorList := ProcessStoredResponses(nil, &rw, fetcher)
+			assert.Equal(t, test.expectedStoredAuctionResponses, storedAuctionResponses, "storedAuctionResponses doesn't match: %s\n", test.description)
+			assert.Equalf(t, test.expectedStoredBidResponses, storedBidResponses, "storedBidResponses doesn't match: %s\n", test.description)
+			assert.Equal(t, test.expectedBidderImpReplaceImpID, bidderImpReplaceImpId, "bidderImpReplaceImpId doesn't match: %s\n", test.description)
+			assert.Nil(t, errorList, "Error should be nil")
```

Comment:
Nitpick: you don't need the error message and test description at the end of these assert statements when using `t.Run`. `test.description` was passed into `t.Run` as the first parameter and if one of the assert statements fails, the description will be automatically be output to give context on the error.